cores/esp8266/core_esp8266_waveform.cpp

```diff
@@ -225,85 +191,97 @@ static ICACHE_RAM_ATTR void timer1Interrupt() {
 
   bool done = false;
   if (waveformEnabled) {
+    uint32_t now = ESP.getCycleCount();
     do {
-      nextEventCycles = microsecondsToClockCycles(MAXIRQUS);
+      nextEventCycle = now + microsecondsToClockCycles(MAXIRQUS);
```

Comment:
Can we assume this `microsecondsToClockCycles(MAXIRQUS)` always yields the same value, or at least as long as this loop is active?
If so, then this value for `nextEventCycle` will always be the same, right?

Reply:
IIRC it's a constexpr :-) At least if the respective one of my various PRs gets applied :-) :-)